todoman/cli.py

```diff
@@ -149,7 +149,7 @@ def new(ctx, summary, list, todo_properties, interactive):
     if not todo.summary:
         raise click.UsageError('No SUMMARY specified')
 
-    default_due = int(ctx.obj['config']['main'].get('default_due'))
+    default_due = ctx.obj['config']['main'].get('default_due')
```

Comment:
`default_due = ctx.obj['config']['main']['default_due']` better stays in line with other usages.